pages/education.py

```diff
@@ -949,16 +946,16 @@ def education_sub_narrative(year, data):
     Input("country-select", "value"),
 )
 def update_education_year_range(data, country):
-    try:
-        data = pd.DataFrame(data["edu_subnational_expenditure"])
-        data = data.loc[(data.func == "Education")]
+    data = pd.DataFrame(data["edu_subnational_expenditure"])
+    data = data.loc[(data.func == "Education")]
 
-        data = filter_country_sort_year(data, country)
-        data = data[data["attendance"].notna()]
-        years = list(data.year.astype("int").unique())
-        years_test = [year for year in years if year % 2 == 0]
-        years.sort()
-        configs = slider_helper(years, years_test)
-        return configs
-    except Exception as e:
-        return {"display": "block"}, {}, 0, 0, 0, {}
\ No newline at end of file
+    data = filter_country_sort_year(data, country)
+    
+    if data.empty:
+        return {"display": "block"}, {}, 0, 0, 0, {}
+
+    expenditure_years = list(data.year.astype("int").unique())
+    data = data[data["attendance"].notna()]
+    outcome_years = list(data.year.astype("int").unique())
+    configs = get_slider_config(expenditure_years, outcome_years)
+    return configs
```

Comment:
Minor: consider skipping the variable assignment to `config` and instead `return get_slider_config(expenditure_years, outcome_years)` directly. Notice the line would be shorter and saves a line of code. We generally want to avoid unnecessary variable assignments.